bin/fix-semantic-css

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
https://www.npmjs.com/package/@semantic-ui-react/css-patch WDYT?

Reply:
In addition, this bash script doesn't cover all cases like BSD OS 😄

Reply:
Thanks for the package, I've committed replacement of my custom script with this one. It seems to work locally, let's wait for CI.